code/modules/surgery/organs/stomach/_stomach.dm

```diff
@@ -288,6 +289,7 @@
 /obj/item/organ/internal/stomach/bone/plasmaman
 	name = "digestive crystal"
 	icon_state = "stomach-p"
+	organ_traits = list()
```

Comment:
Do plasmamen get hungry now, then?

Reply:
Their lungs give them the NOHUNGER trait. They feed off plasma.

Reply:
And also from a balance perspective, it would be super over powered if plasma men had a super stomach that prevented the need for hunger that could be implanted in others. This way their lack of need to eat is tied in to the constraint of having to breathe plasma.